src/cmds/pbs_rsub.c

```diff
@@ -143,6 +144,7 @@ process_opts(int argc, char **argv, struct attrl **attrp, char *dest)
 				break;
 
 			case 'l':
+				opt_res_req_flg = TRUE;
```

Comment:
@vchlum my comment was about option -I (ATTR_inter) but this works too because --hosts will consume all ncpus on the hosts anyways. Can you please add similar change to "ATTR_inter" option as well?